conf/cb-event-forwarder.example.ini

```diff
@@ -385,3 +395,18 @@ events_binary_upload=ALL
 #  (used in HTTP Basic Authentication). See https://en.wikipedia.org/wiki/Basic_access_authentication
 #  for more information. By default no Authorization header is sent.
 # authorization_token=Basic QWxhZGRpbjpPcGVuU2VzYW1l
+
+[kafka]
+# Uncomment when using kafka brokers. Broker configuration should be a comma-separated
+# list of brokers with ports.
+# e.g.
+# brokers=kakfa01:9092,kafka02:9092
+# It is not necessary to put all of your brokers here, just enough to dependably
+# bootstrap the producer
+
+# Optional additional suffix to add to the name of the topics.
+# By default the topic published to will be the type of the event, e.g. "procstart'
+# Must be uncommented when using kafka output but can be left blank
+# topic_suffix=-test
+[splunk]
+hec_token=PASSWORD
```

Comment:
copy the HTTP options into the splunk section - TLS verification, etc. to show you can use those in the splunk output as well